### PR TITLE
Update organisation dao model rest

### PR DESCRIPTION
**Pull request by kentsanggds**
Updated Organisations model, and DAO to allow creating and updating of Organisations

Update of Organisation model to make logo not nullable
- migration script is included as part of this PR.

Also refactored code in organisation rest to remove marshmallow code and use a json_schema instead.

Part 1 / 4 - https://www.pivotaltracker.com/story/show/148289053

### CODE REVIEW

File: tests/app/organisation/test_rest.py [hunk 1]
```diff
@@ -37,3 +37,52 @@ def test_get_organisation_by_id(notify_api, notify_db, notify_db_session):
...
+
+
+def test_create_organisation_without_logo_raises_error(client, notify_db, notify_db_session):
+    data = {
+        'name': 'test organisation',
+        'colour': '#0000ff',
+    }
+    auth_header = create_authorization_header()
+
+    response = client.post(
```
Comment (quis) [inline tests/app/organisation/test_rest.py]: This could be cleaner if you used the `admin_request` fixture that Leo made. For example:

```python
response = admin_request.post(
  '/organisation',
  data={
    'name': 'test organisation',
    'colour': '#0000ff',
  }
  _expected_status=400
)
  
assert response == {…}
```

Comment (quis) [inline tests/app/organisation/test_rest.py]: Same for all these tests.

Comment (kentsanggds) [inline tests/app/organisation/test_rest.py]: ok, sure

File: app/dao/organisations_dao.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+def dao_get_organisations():
+    return Organisation.query.all()
+
+
+def dao_get_organisation_by_id(org_id):
+    return Organisation.query.filter_by(id=org_id).one()
+
+
+@transactional
+def dao_create_organisation(organisation):
```
Comment (imdadahad) [inline app/dao/organisations_dao.py]: Hmm, this should take some kwargs like other dao methods do. Otherwise it's just doing the same as `dao_update_organisation` when it probably should be doing more.

File: app/dao/organisations_dao.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from app import db
```
Comment (servingUpAces) [inline app/dao/organisations_dao.py]: Why was this file renamed?

File: migrations/versions/0107_change_logo_not_nullable.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+"""empty message
```
Comment (servingUpAces) [inline migrations/versions/0107_change_logo_not_nullable.py]: You'll need to rename this file now - there is a new db migration.

Comment (kentsanggds) [inline migrations/versions/0107_change_logo_not_nullable.py]: ok

File: app/dao/organisations_dao.py (hunk 3 above)
Comment (kentsanggds) [inline app/dao/organisations_dao.py]: trying to keep it in line with other daos, if it's an issue I can change it back
